src/Kestrel.Https/baseline.netcore.json

```diff
@@ -1,5 +1,5 @@
 {
-  "AssemblyIdentity": "Microsoft.AspNetCore.Server.Kestrel.Https, Version=2.0.2.0, Culture=neutral, PublicKeyToken=adb9793829ddae60",
+  "AssemblyIdentity": "Microsoft.AspNetCore.Server.Kestrel.Https, Version=2.1.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60",
```

Comment:
@Eilon @ryanbrandenburg I'm using 2.1.0 as the baseline, though in other repos I'm seeing a different base versions. For example, Mvc uses 2.1.1: https://github.com/aspnet/Mvc/blob/release/2.2/src/Microsoft.AspNetCore.Mvc/baseline.netcore.json#L2. Which version should we be using?

Reply:
If we know the baseline is exactly the same for 2.1.0 and 2.1.3 as it theoretically should be, I would say using 2.1.0 as the baseline version is better so we're not inclined to update the baseline version with every patch.

Do we automatically verify that we don't add new APIs in patch releases? If not, I can see the argument for using the latest patch to ensure we don't break said new APIs in the future.

Reply:
Doesn't matter among 2.1.x because the APIs should be identical within those.  Do whatever is easiest for now.

Reply:
I think 2.1.0 is sufficient for now then.